profiles/lsf/config.yaml

```diff
@@ -0,0 +1,20 @@
+reason: true
+rerun-incomplete: true
+keep-going: true
+jobs: 500
+max-jobs-per-second: 1
+use-conda: true
+conda-frontend: mamba
+latency-wait: 120
+cluster: bsub -cwd 
+              -q {resources.partition}
+              -n {threads}
+              -o {resources.out}
+              -e {resources.err} {resources.extra}
+default-resources: 
+  - partition='default'
+  - tmpdir=system_tmpdir
```

Comment:
Does this get passed through correctly to containers?

Reply:
Also, the use of tmpdir in --default-resources requires a newer version of snakemake (>=6.5.2), but I tested on a more recent version so the environment.yaml was updated to reflect the testing version (>=6.15).